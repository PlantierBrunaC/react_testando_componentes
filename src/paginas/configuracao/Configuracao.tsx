import Formulario from "../../componentes/Formulario";
import IniciarSorteio from "../../componentes/IniciarSorteio/IniciarSorteio";
import ListaParticipantes from "../../componentes/ListaParticipantes/ListaParticipantes";


const Configuracao = () => {
return (
    <>
    <Formulario/>
    <ListaParticipantes/>
    <IniciarSorteio/> 
    </>

)

}
export default Configuracao;