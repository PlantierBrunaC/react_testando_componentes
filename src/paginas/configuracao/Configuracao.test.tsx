import { render } from "@testing-library/react"
import { RecoilRoot } from "recoil"
import Configuracao from "./Configuracao"

const mockNavegacao = jest.fn()
jest.mock('react-router-dom', () =>{
    return {
        useNavigate: () => mockNavegacao
    }
})


describe('Página de configuracao', () => {

    test('Página deve ser renderizada corretamente', () => {
        const { container } = render(
        <RecoilRoot>
            <Configuracao /> 
            </RecoilRoot>)
            expect(container).toMatchSnapshot()
    

    })

})