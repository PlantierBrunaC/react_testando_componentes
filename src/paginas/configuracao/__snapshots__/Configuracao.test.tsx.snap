// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Página de configuracao Página deve ser renderizada corretamente 1`] = `
<div>
  <form>
    <h1>
      Vamos começar !
    </h1>
    <input
      placeholder="Insira os nomes dos participantes"
      type="text"
      value=""
    />
    <button
      disabled=""
    >
      Adicionar
    </button>
    
  </form>
  <ul />
  <button
    disabled=""
  >
    Iniciar Brincadeira!
  </button>
</div>
`;
